getting_started/step_by_step/1_create_escoria_project.rst

```diff
@@ -39,6 +39,8 @@ Click on **Download** to download a fresh copy of the template, then click
 
 Enter the project name and the folder for your new game.
 Click **Install & Edit**.
+Note that if you launched Godot from a terminal then this terminal will display
```

Comment:
I'd suggest putting the text into the warning section that is about the same thing basically. Or maybe move the warning up to the position you added the text if it makes more sense there.

Reply:
To me it wasn't clear at all this was about the same thing. I asked on Discord and @StraToN told me to simply ignore these messages. So maybe you should talk together to decide whether it can be ignored or it means people should "quit to the project manager and reopen the game to let Escoria initialize properly." I don't have any opinion here, I launched Godot for the first time today. Also feel free to edit my text if you want.

Reply:
@StraToN I believe the error messages are due to the import process and initialization of three plugins. Restarting Godot should fix that so we maybe should add @PatrickMassot 's text to the warning and move it a bit up. Do you agree?

Reply:
Yes I agree with that. I tried multiple times to get these error messages but they don't seem to appear every time.